src/main/java/Luke.java

```diff
@@ -0,0 +1,246 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+public class Luke {
+
+    static String[] validCommands = {"bye", "list", "unmark", "mark", "todo", "event", "deadline", "delete"};
+
+    static String directoryPath = "./data";
+
+    static String fileName = "list.txt";
+
+    static File file = new File (directoryPath, fileName);
```

Comment:
Missing visiblity for these variables. Should they also be declared as constants?
Unnecessary blank lines between each declaration.